source/crud/read-operations/cursor.txt

```diff
@@ -0,0 +1,150 @@
+=========================
+Access Data From a Cursor
+=========================
+
+.. default-domain:: mongodb
+
+Read operations that return multiple documents do not immediately return
+all values matching the query. Because a query can potentially match
+very large sets of documents, these operations rely upon an
+object called a **cursor** that fetches documents in batches to reduce
+both memory consumption and network bandwidth usage. Cursors
+are highly configurable and offer multiple interaction paradigms for
+different use cases.
+
+The following functions directly return cursors:
+
+- ``Collection.find()``
+
+- ``Collection.aggregate()``
+
+- ``Collection.listIndexes()``
+
+- ``Db.aggregate()``
+
+- ``Db.listCollections()``
+
+Other methods, like ``Collection.findOne()`` and ``Collection.watch()``
```

Comment:
Good point, `generate` probably isn't the right word choice here. I'm going to change this to `return` because that is probably a more familiar term to many readers and is technically correct (though it does gloss over the details... but we get to those later anyway). Going to avoid using the term "private" in this case because javascript doesn't actually have a formal notion of private like some other languages and I feel like the current wording is a little less jargon-y.